lang/nodejs/monitoring.yml

```diff
@@ -0,0 +1,6 @@
+releases:
+  id: 12594
+security:
+  cpe:
+  - vendor: nodejs
```

Comment:
So you should probably add the other CPE entry as well. `nodejs:nodejs`. It won't hurt just in case a CVE gets miscategorized or something.